source/docs/3/advanced_raster_analysis.rst

```diff
@@ -0,0 +1,176 @@
+
+  .. code-block:: none
+
+     "SA_NLC_2018_Gauteng@1" >= 51 AND "SA_NLC_2018_Gauteng@1" <= 54
+
+
+  .. image:: /static/3/advanced_raster_analysis/images/05.png
+    :align: center
+
+6. Scroll down and click the ``...`` button next to :guilabel:`Reference layer(s)`. Select the ``SA_NLC_2018_Gauteng`` layer and click :guilabel:`OK`.
+
+  .. image:: /static/3/advanced_raster_analysis/images/06.png
+    :align: center
+
+7. Next, click the ``...`` button next to :guilabel:`Output` and select :guilabel:`Save to File...`.
+
+  .. image:: /static/3/advanced_raster_analysis/images/07.png
+    :align: center
+
+8. Name the output file ``residential_informal.tif`` and click :guilabel:`Run`.
+
+  .. image:: /static/3/advanced_raster_analysis/images/08.png
+    :align: center
+
+9. Once the processing finishes, a new layer ``residential_informal`` will be added to QGIS. This raster layer has only 2 pixel values - ``1`` where our expression evaluated to *true* and ``0`` where it was *false*. The pixels that appear white are the ones belonging to the informal settlement classes.
+
+  .. image:: /static/3/advanced_raster_analysis/images/09.png
+    :align: center
+
+10. We will now style this layer better so we can see the informal settlements clearly. Turn off the ``SA_NLC_2018_Gauteng`` layer from the :guilabel:`Layers` panel. Select the ``residential_informal`` layer and click the :guilabel:`Open the layer styling panel` button. Change the renderer to be :guilabel:`Paletted/Unique values`.
+
+  .. image:: /static/3/advanced_raster_analysis/images/10.png
+    :align: center
+
+11. Click the :guilabel:`Add values manually` (+) button. Change the :guilabel:`Value` to ``1`` and enter ``Residential Informal`` as :guilabel:`Label`. Select a color of your choice.
+
+  .. image:: /static/3/advanced_raster_analysis/images/11.png
+    :align: center
+
+12. We can now see all the informal settlement in the Gauteng region. It would be helpful to see them in context with a basemap. We have access to a variety of base-maps from the :guilabel:`QuickMapServices` plugin. Once you install the plugin, go to :menuselection:`Web --> QuickMapServices --> OSM --> OSM Standard` to add the OpenStreetMap layer.
+
+  .. image:: /static/3/advanced_raster_analysis/images/12.png
+    :align: center
+
+13. Now you can easily identify and verify whether our analysis correctly identified the informal settlements. You can select the ``residential_informal`` layer and switch to the :guilabel:`Transparency` tab in the :guilabel:`Layer styling panel`. You can reduce the :guilabel:`Global Opacity` to see both the extracted pixels and the basemap together.
+
+  .. image:: /static/3/advanced_raster_analysis/images/13.png
+    :align: center
+
+14. To identify the urban growth pattern in all the areas where the land cover changed to the built-up area from 2014 to 2018. Drag and drop the ``SA_NLC_2014_2018_CHANGE_Gauteng.tif`` file from the *Browser* to the canvas. 
```

Comment:
This is a completely difference exercise with different dataset. So give a transition saying we are done with part-1 of the exercise and we will start the next part with a different dataset.